crypto/fipsmodule/pqdsa/pqdsa.c

```diff
@@ -109,10 +118,13 @@ int PQDSA_KEY_set_raw_keypair_from_seed(PQDSA_KEY *key, CBS *in) {
     return 0;
   }
 
+  // copy the seed data
+  OPENSSL_memcpy(seed, CBS_data(in), key->pqdsa->keygen_seed_len);
+
   // set the public and private key
   key->public_key = public_key;
   key->private_key = private_key;
-
+  key->seed = seed;
```

Comment:
Andrew I know you're going to be reading this and thinking that I should just let `key` take ownership of the memory and assign directly like `key->CBS_data(in)` but `CBS_data(in)` is a `const` so it fails. We don't want `seed` to be `const` in `int ml_dsa_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk, uint8_t *seed);` because `seed` is input as an empty buffer of allocated `ML_DSA_SEEDBYTES` bytes, and is output as the seed.